imports/plugins/included/search-mongo/server/methods/searchcollections.js

```diff
@@ -167,36 +167,52 @@ export function buildOrderSearchRecord(orderId) {
       orderSearch[field] = order[field];
     }
   }
-  orderSearch.billingName = order.billing[0].address.fullName;
-  orderSearch.billingPhone = _.replace(order.billing[0].address.phone, /\D/g, "");
-  orderSearch.shippingName = order.shipping[0].address.fullName;
-  orderSearch.shippingPhone = _.replace(order.shipping[0].address.phone, /\D/g, "");
+  // get the billing object for the current shop on the order (and not hardcoded [0])
+  const shopBilling = order.billing.find(
+    billing => billing.shopId === Reaction.getShopId()
+  ) || {};
+
+  // get the shipping object for the current shop on the order
+  const shopShipping = order.shipping.find(
+    shipping => shipping.shopId === Reaction.getShopId()
```

Comment:
I don't think this will work as the shipping record currently doesn't contain a shopId? Maybe we just fly with this for now since that should be added shortly, idk.

Reply:
Or I can add leave as before on the zeroth index [0]; and add a TODO for when shipping update happens.